rpc/wrtc_base_channel.go

```diff
@@ -174,8 +181,19 @@ func (ch *webrtcBaseChannel) onChannelClose() {
 	}
 }
 
+// isUserInitiatedAbortChunkErr returns true if the error is an abort chunk
+// error that the user initiated through Close. Certain browsers (Safari,
+// Chrome and potentially others) close RTCPeerConnections with this type of
+// abort chunk that is not indicative of an actual state of error.
+func isUserInitiatedAbortChunkErr(err error) bool {
+	return err != nil && errors.Is(err, sctp.ErrChunk) &&
+		strings.Contains(err.Error(), "User Initiated Abort: Close called")
```

Comment:
for later, it may be nice to export the error in https://github.com/pion/sctp/blob/d6446e3b0d0495d9974e68fec9b7fdb79c018ca1/error_cause_user_initiated_abort.go#LL25C6-L25C34 or to provide a function check if it is such an error

Reply:
> or to provide a function check if it is such an error

yeah good idea, I'll open a PR to try to add a function. it's not exactly an "error" and more an _error cause_ that gets marshaled and included in the list of error causes for the chunk (and a packet is a slice of chunks), so I don't know if exporting that `errorCauseUserInitiatedAbort` type will help.